tideways-daemon.pkr.hcl

```
@@ -70,12 +70,16 @@ build {
     "source.amazon-ebs.tideways-daemon"
   ]
 
+  provisioner "file" {
+    source="tideways.source"
+    destination="/tmp/tideways.source"
+  }
+
   provisioner "shell" {
     inline = [
       # Install the daemon
       # See https://support.tideways.com/documentation/setup/installation/debian-ubuntu.html
-      "echo 'deb https://packages.tideways.com/apt-packages-main any-version main' | sudo tee /etc/apt/sources.list.d/tideways.list",
-      "wget -qO - https://packages.tideways.com/key.gpg | sudo apt-key add -",
+      "mv /tmp/tideways.source /etc/apt/sources.list.d./tideways.source",
       "sudo apt update",
       "sudo apt install -y tideways-daemon",
```

The `apt` frontend is not meant for scripting:

```suggestion
      "sudo apt-get update",
      "sudo apt-get install -y tideways-daemon",
```